### ISSUE TITLE
Stacking context not affected by isolation

### ISSUE THREAD

**Issue by Mire** [w3c/csswg-test#376]
## Test case description

2 elements required:
[P] and
[B] 
[P] - parent element with a property that creates a stacking context (e.g. position:fixed); This element has isolation explicitly set to auto 
[B] - element with mix-blend-mode other than normal 
The blending element [B] has content that lies outside the parent element. 
Set the background-color of the body to a value other than default

For details about the notations and more in-context information please refer to section **An element with isolation:auto set does not change the elements existing stacking context behavior** of the test plan:
http://mire.github.io/css-blending-test-plan-proposal/css-blending-test-plan-proposal.html
## Expected result

The color of the parent element mixes with the color of the child element. 
The area of the child element outside of the parent element doesn't mix with the color of the body.
In other words, setting the isolation to auto does not affect the creation of a stacking context by other properties.
## W3C specification

This test case pertains to the following w3c specification:
http://www.w3.org/TR/compositing-1/

**Comment by wpt-issue-mover** [comment]
This issue has been merged into https://github.com/w3c/web-platform-tests/issues/5555; please see that issue if you wish to discuss or work on this.
